src/RelayClient.ts

```diff
@@ -809,14 +811,16 @@ export class RelayClient {
                 gasPrice,
                 callVerifier,
                 callForwarder: forwarderAddress,
-                relayWorker
+                feesReceiver: collectorAddress
```

Comment:
Would not be better to standardise the name to feeesReceiver?

Reply:
Had exactly the same thought, but we might keep the distinction for explicitness. After all, the fees receiver is not always the collector contract.

Reply:
Ended up uses feesReceiver after all